nservicebus/service-fabric/transaction-sharing.md

```diff
@@ -18,6 +18,12 @@ include: saga-business-data-access
 
 snippet: ServiceFabricPersistenceSynchronizedSession-Saga
 
-Warning: Shared transaction should not be committed (`CommitAsync()`) or disposed (`Dispose()`) in the handler or the saga. When data needs to be persisted to a custom collection during handler or saga execution, a new transaction should be created using the `StateManager` property of the incoming session.
+Warning: Shared transaction should not be committed (`CommitAsync()`) or disposed (`Dispose()`) in the handler or the saga. 
+
+### Using a custom transaction
+
+When data needs to be persisted to a custom collection during handler or saga execution, a new transaction should be created using the `StateManager` property of the incoming session.
```

Comment:
Not sure this is 100% correct. Isn't the answer it depends? If you want to participate in the same transactions as sagas use the transaction prop provided, if you want tx but that doesn't need to be connected to owned tx use your own. Isn't that what we want to recommend?